modules/SilkSpawners/src/main/java/de/dustplanet/silkspawners/listeners/SilkSpawnersBlockListener.java

```diff
@@ -240,7 +240,6 @@ public void onBlockPlace(BlockPlaceEvent event) {
         plugin.getServer().getPluginManager().callEvent(placeEvent);
         // See if we need to stop
         if (placeEvent.isCancelled()) {
-            event.setCancelled(true);
```

Comment:
This won't solve the underlying issue - this is a feature of the custom SilkSpawnersSpawnerPlaceEvent and this "upstream" BlockPlaceEvent has to be cancelled when the SilkSpawnersSpawnerPlaceEvent was cancelled by any consumer (e.g. SilkSpawnersEcoAddon)